.github/workflows/scripts/push_branch_and_tag_to_github.sh

```diff
@@ -7,7 +7,7 @@ ref_string=$(git show-ref --tags | grep refs/tags/$1)
 
 SHA=${ref_string:0:40}
 
-remote_repo=https://ansible:${RELEASE_TOKEN}@github.com/${GITHUB_REPOSITORY}.git
+remote_repo=https://pulpbot:${RELEASE_TOKEN}@github.com/${GITHUB_REPOSITORY}.git
```

Comment:
I think you got the wrong user

Reply:
We don´t have yet an `galaxybot` user, the token should be created for the specific user? maybe we need to create that user and add to the org.

Reply:
not sure, as we have pulpbot we always used it, maybe other user will work, but I never tested